contribs/src/main/java/com/netflix/conductor/contribs/queue/nats/config/NATSStreamConfiguration.java

```diff
@@ -28,4 +36,33 @@
     public EventQueueProvider natsEventQueueProvider(NATSStreamProperties properties, Scheduler scheduler) {
         return new NATSStreamEventQueueProvider(properties, scheduler);
     }
+
+    @ConditionalOnProperty(name = "conductor.default-event-queue.type", havingValue = "nats_stream", matchIfMissing = true)
```

Comment:
Please remove the `match_if_missing=true` part. The default value is set to `sqs` and we wouldn't want to change that.

Reply:
done